packages/variable-editor/src/VariableEditor.tsx

```diff
@@ -60,14 +60,16 @@ function VariableEditor(props: DataContext) {
     return <p>{'An error has occurred: ' + error}</p>;
   }
 
-  const variables = toVariables(data.data);
+  const rootVariable = toVariables(data.data);
```

Comment:
How many times have you renamed this variable from 'variables' to 'rootVariable' and back again? ;-)

Reply:
I don't think I have, maybe you mean another one? :sweat_smile: